src/bindings/overlay.js

```diff
@@ -130,7 +141,10 @@ function overlay(sourceElement, translationElement) {
   // cleared if a new language doesn't use them; https://bugzil.la/922577
   if (translationElement.attributes) {
     for (k = 0, attr; (attr = translationElement.attributes[k]); k++) {
-      if (isAttrAllowed(attr, sourceElement)) {
+      if (isAttrAllowed({
```

Comment:
`isAttrAllowed` is meant to accept a DOM Attribute-like object, similar to `isElementAllowed` accepting a DOM Element.  I'd like to keep it that way, so I'd suggest to just pass `attr` here, which already has the `name` and the `namespaceURI` properties.

Also, this code shouldn't just take `translationElement`'s namespace as the attribute's namespace. Given an XHTML document and a translation value of `foo <span xml:lang="en">bar</span>!`, the `span` element's namespace is `http://www.w3.org/1999/xhtml` but the `lang`'s attribute namespace is `http://www.w3.org/XML/1998/namespace`.